index.html

```diff
@@ -58,7 +60,221 @@ <h3>
 
 </section>
    
+
+<header class="title-header">
+    <h2 class="title">My Recent Works</h2>
+  </header>
+  <section class="recent-works">
+      <div class="work">
+          <section >
```

Comment:
- [ ] I think it's better to move the header with the title My recent works inside the recent-works section. Because it belongs to that section.